src/components/SingleArticle.jsx

```diff
@@ -65,19 +76,19 @@ function SingleArticle() {
 
 }
 
+// move to a seperate file
+function DisplayCommentForm ({children}) {
+    const [form, setForm] = useState(false);
 
-function DisplayComments ({children}) {
-    const [showComments, setShowComments] = useState(false);
-
-    const toggleShow = () => {
-        setShowComments((currState) => !currState)     
+    const toggleForm = () => {
+        setForm((currState) => !currState)     
     }
 
     return <div>
-            {showComments ? children : null}
-            {console.log(showComments)}
-            <button onClick={toggleShow}> Show Comments </button>
-    </div>   
-}
+        {form ? children : null}
+        <button onClick={toggleForm}> add comment </button>
+    </div>
```

Comment:
Similar here to the Show Comments button, when you click the add comment button the form is revealed but then you can click the button again to hide the form when the button still just says add comment :)